### ISSUE TITLE
Anti-PM-Scl antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6596]
**Preferred term label:** Anti-PM-Scl antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anti-PM-Scl antibodies target components of  RNA-processing exosome complex in the nucleolus. There are ten proteins in this complex and antibodies to eight of them are found at varying frequencies; PM/Scl-100 (70–80%), PM/Scl-75 (46–80%), hRrp4 (50%), hRrp42 (21%), hRrp46 (18%), hCs14 (14%), hRrp41 (10%) and hRrp40 (7%)
Anti-PM-Scl antibodies are found in up to 50% of polymyositis/systemic sclerosis (PM/SSc) overlap syndrome. Around 80% of individuals with antibodies present in their blood serum will have the disorder. The presence of the antibodies is linked to limited cutaneous involvement of PM/SSc overlap syndrome. PMC 165038,  PMID 17643929

**Parent term (use hpo.jax.org/app)**
Antinuclear antibody positivity HP:0003493

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Limited cutaneous PM/SSc overlap syndrome

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
